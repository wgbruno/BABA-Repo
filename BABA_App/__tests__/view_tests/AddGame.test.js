import {expect, jest, test} from '@jest/globals';
import renderer from 'react-test-renderer';
import CreateGame from '../../View/AddGame';

test("New Game Screen", function () {
    //const newG = CreateGame;
    const tree = renderer.create(<CreateGame/>).toJSON();
    expect(tree).toMatchSnapshot();
});