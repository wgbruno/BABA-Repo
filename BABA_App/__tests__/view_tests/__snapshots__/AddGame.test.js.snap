// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`New Game Screen 1`] = `
<RCTScrollView
  showsVerticalScrollIndicator={false}
>
  <View>
    <View
      style={
        Object {
          "alignItems": "center",
          "padding": 20,
        }
      }
    >
      <Image
        resizeMode="contain"
        source={
          Object {
            "testUri": "../../../assets/images/Logo_1.png",
          }
        }
        style={
          Array [
            undefined,
            Object {
              "height": 26680,
            },
          ]
        }
      />
      <Text
        style={
          Object {
            "color": "darkblue",
            "fontSize": 40,
            "fontWeight": "bold",
            "marginVertical": 10,
          }
        }
      >
        Schedule a new game
      </Text>
      <View
        style={
          Object {
            "backgroundColor": "white",
            "borderColor": "#e8e8e8",
            "borderRadius": 5,
            "borderWidth": 1,
            "marginVertical": 5,
            "paddingHorizontal": 10,
            "width": "100%",
          }
        }
      >
        <TextInput
          onChangeText={[Function]}
          placeholder="Team Name 1"
          style={Object {}}
          value=""
        />
      </View>
      <View
        style={
          Object {
            "backgroundColor": "white",
            "borderColor": "#e8e8e8",
            "borderRadius": 5,
            "borderWidth": 1,
            "marginVertical": 5,
            "paddingHorizontal": 10,
            "width": "100%",
          }
        }
      >
        <TextInput
          onChangeText={[Function]}
          placeholder="Team Name 2"
          style={Object {}}
          value=""
        />
      </View>
      <View
        style={
          Object {
            "backgroundColor": "white",
            "borderColor": "#e8e8e8",
            "borderRadius": 5,
            "borderWidth": 1,
            "marginVertical": 5,
            "paddingHorizontal": 10,
            "width": "100%",
          }
        }
      >
        <TextInput
          onChangeText={[Function]}
          placeholder="Start Time"
          style={Object {}}
          value=""
        />
      </View>
      <View
        style={
          Object {
            "backgroundColor": "white",
            "borderColor": "#e8e8e8",
            "borderRadius": 5,
            "borderWidth": 1,
            "marginVertical": 5,
            "paddingHorizontal": 10,
            "width": "100%",
          }
        }
      >
        <TextInput
          onChangeText={[Function]}
          placeholder="Date"
          style={Object {}}
          value=""
        />
      </View>
      <View
        accessible={true}
        collapsable={false}
        focusable={true}
        onBlur={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Array [
            Object {
              "alignItems": "center",
              "borderRadius": 5,
              "marginVertical": 5,
              "padding": 10,
              "width": "100%",
            },
            Object {
              "backgroundColor": "#DD4D44",
            },
            Object {},
          ]
        }
      >
        <Text
          style={
            Array [
              Object {
                "color": "white",
                "fontWeight": "bold",
              },
              undefined,
              Object {},
            ]
          }
        >
          Create Game
        </Text>
      </View>
    </View>
  </View>
</RCTScrollView>
`;
